fido/src/main/java/com/yubico/yubikit/fido/client/BasicWebAuthnClient.java

```diff
@@ -476,14 +470,14 @@ protected Ctap2Session.CredentialData ctapMakeCredential(
                     getCredentialList(excludeCredentials),
                     null,
                     ctapOptions.isEmpty() ? null : ctapOptions,
-                    pinUvAuthParam,
-                    pinUvAuthProtocol,
+                    authParams.pinUvAuthParam != null ? authParams.pinUvAuthParam.clone() : null,
+                    authParams.pinUvAuthProtocol,
                     validatedEnterpriseAttestation,
                     state
             );
         } finally {
-            if (pinToken != null) {
-                Arrays.fill(pinToken, (byte) 0);
+            if (authParams.pinUvAuthParam != null) {
+                Arrays.fill(authParams.pinUvAuthParam, (byte) 0);
             }
```

Comment:
I don't think we need to consider authParam as sensitive data. It's basically a signature over the command, it doesn't contain any sensitive info in itself.